p/pecl-encryption-mcrypt/pecl-encryption-mcrypt_ubi8.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: pecl-encryption-mcrypt
+# Version	: 1.0.3
+# Source repo	: https://github.com/php/pecl-encryption-mcrypt
+# Tested on	: UBI 8.5
+# Language	: PHP
+# Travis-Check	: True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vedang Wartikar <Vedang.Wartikar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=pecl-encryption-mcrypt
+PACKAGE_VERSION=1.0.3
```

Comment:
Can you make PACAKGE_VERSION as configurable ?
Something like : 
`PACKAGE_VERSION=${1:-1.0.3}`